app/models/activity_insight_oa_file.rb

```diff
@@ -58,6 +69,8 @@ def version_status_display
           bindings[:view].link_to("Download #{bindings[:object].download_filename}", Rails.application.routes.url_helpers.activity_insight_oa_workflow_file_download_path(bindings[:object].id))
         end
       end
+      field(:downloaded)
+      field(:download_location_value) { label 'File download' }
```

Comment:
Could we consolidate the 'File' and 'File download' field?  Maybe just have a single 'File download' field with this in it:

`bindings[:view].link_to("#{bindings[:object].download_location_value}", Rails.application.routes.url_helpers.activity_insight_oa_workflow_file_download_path(bindings[:object].id))`

Reply:
Yeah, good call.